Personal/jebearer/lts (#2832)
MicrosoftDocs/azure-stack-docs: azure-managed-lustre/includes/client-install-ubuntu-22.md

---

> [!IMPORTANT]
> The Azure Marketplace image for the Ubuntu 22.04 LTS release uses the Hardware Enablement (HWE) Kernel by default. However, these kernels are only supported for 6 month periods, and Lustre support for these kernels is often not available when they are released. Therefore, it is recommended that you switch to the LTS kernel, as it will give you more stability as well as maintaining a kernel version that is supported with the Lustre Client.

1. Install the LTS Kernel Metapackage.

   ```bash
   sudo apt update && sudo apt install linux-image-azure-lts-22.04
   ```

1. Remove the HWE Kernel Metapackage.

>    Remove the default (Hardware Enablement) Kernel metapackage, it will also ask to remove the linux-azure metapackage.  This is expected.

Reviewer: ```suggestion
   Remove the default (Hardware Enablement) Kernel metapackage. It will also ask to remove the linux-azure metapackage.  This is expected.
```

   ```bash
   sudo apt remove linux-image-azure
   ```

> 1. List installed kernels and see which one is supplied by the LTS metapackage

Reviewer: ```suggestion
1. List installed kernels and see which one is supplied by the LTS metapackage.
```

>    After the metadata package is removed check to see what kernels are currently installed.  Newly provisioned hosts will have 2 kernels, older hosts may have more. Compare the version that the LTS metapackage provides against the other installed kernels.  Here you will see that a 6.2 kernel is still installed previously from the linux-image-azure metapackage.

Reviewer: ```suggestion
   After the metadata package is removed, check to see what kernels are currently installed.  Newly provisioned hosts will have two kernels and older hosts might have more. Compare the version that the LTS metapackage provides against the other installed kernels.  Here you see that a 6.2 kernel is still installed previously from the linux-image-azure metapackage.
```

   ```bash
   apt list --installed linux-image*
   ```

> 1. Remove any kernels newer than the one mentioned in the LTS metapackage

Reviewer: ```suggestion
1. Remove any kernels newer than the one mentioned in the LTS metapackage.
```

>    Remove any kernels other than the one mentioned in the LTS kernel metapackage.  It will warn about removing the kernel and recommending aborting the process. If you're folowing these steps on a newly provisioned host these work. But if you have concerns, consult ubuntu documentation on configuring kernels to ensure it's able to boot after a reboot.

Reviewer: ```suggestion
   Remove any kernels other than the one mentioned in the LTS kernel metapackage.  It will warn about removing the kernel and recommend aborting the process. If you're following these steps on a newly provisioned host, they work. But if you have concerns, consult Ubuntu documentation on configuring kernels to ensure it's able to boot after a reboot.
```
